packages/flutter/lib/src/cupertino/route.dart

```diff
@@ -237,7 +237,7 @@ class CupertinoPageRoute<T> extends PageRoute<T> {
 
   @override
   Widget buildPage(BuildContext context, Animation<double> animation, Animation<double> secondaryAnimation) {
-    final Widget result = builder(context);
+    final Widget result = new Semantics(route: true, explicitChildNodes: true, child: builder(context));
```

Comment:
nit: formatting here and in many cases below: one line per property to make it easier to read?

Reply:
done